packages/templates/api/assemblyscript/package.json

```diff
@@ -15,11 +15,17 @@
     "test": "npx w3 query ./recipes/e2e.json"
   },
   "devDependencies": {
+    "@types/jest": "^27.0.3",
```

Comment:
Dependencies should be locked. Please remove the `^` version resolution modifier.